packages/ui/src/SmartResourcePropertyDisplay.tsx

```diff
@@ -0,0 +1,19 @@
+import { PropertyType } from '@medplum/core';
+import { Resource } from '@medplum/fhirtypes';
+import { evalFhirPath } from '@medplum/fhirpath';
+import React from 'react';
+import { ResourcePropertyDisplay } from '.';
+
+export interface SmartResourcePropertyDisplayProps {
+  resource: Resource;
+  path: string;
+}
+
+export function SmartResourcePropertyDisplay(props: SmartResourcePropertyDisplayProps): JSX.Element {
+  const value = evalFhirPath(props.path, props.resource);
+
+  if (value?.[0] && value.length === 1) {
```

Comment:
Thoughts on quietly accepting multiple values?

Reply:
I believe `evalFhirPath` always returns an array, so you should be able to remove the first clause.

I.e., this should be ok:

`if (value.length === 1) {`

Reply:
> Thoughts on quietly accepting multiple values?

Tell me if this is bad practice, but I think it makes sense to be explicit about our assumption that the path only returns one value, rather than just taking the first. Thinking it might make sense to throw an error if `length !== 1` so that gets feedback on the behavior

Reply:
Pros/cons, I'm happy to go either way.  In the short term, I think we expect to hand craft these, so we're building for ourselves.